contrib/fsl_integration/sct_plugin.py

```diff
@@ -208,52 +290,703 @@ def __init__(self, parent):
+        bvecname_path = os.path.join(path_name, bvec_name)
+
+        print('Bvec file:', bvecname_path)
+
+        bval_name = "{}_bvals.txt".format(fname)
+        bvalname_path = os.path.join(path_name, bval_name)
+
+        print('Bval file:', bvalname_path)
+
+        if outname == '':
+
+            print('Output file name not defined, standard name will be used.')
+            fa_name = "dti_FA.nii.gz"
+            faname_path = os.path.join(os.getcwd(), fa_name)
+
+            ad_name = "dti_AD.nii.gz"
+            adname_path = os.path.join(os.getcwd(), ad_name)
+
+            md_name = "dti_MD.nii.gz"
+            mdname_path = os.path.join(os.getcwd(), md_name)
+
+            rd_name = "dti_RD.nii.gz"
+            rdname_path = os.path.join(os.getcwd(), rd_name)
+
+            cmd_line = "sct_dmri_compute_dti -i {} -bval {} -bvec {}".format(rawimg, bvalname_path,
+                                                                             bvecname_path)
+            print('Command line:', cmd_line)
+
+        else:
+
+            print('Output file name:', outname)
+            fa_name = "{}FA.nii.gz".format(outname)
+            faname_path = os.path.join(os.getcwd(), fa_name)
+
+            ad_name = "{}AD.nii.gz".format(outname)
+            adname_path = os.path.join(os.getcwd(), ad_name)
+
+            md_name = "{}MD.nii.gz".format(outname)
+            mdname_path = os.path.join(os.getcwd(), md_name)
+
+            rd_name = "{}RD.nii.gz".format(outname)
+            rdname_path = os.path.join(os.getcwd(), rd_name)
+
+            cmd_line = "sct_dmri_compute_dti -i {} -bval {} -bvec {} -o {}".format(rawimg, bvalname_path, bvecname_path, outname)
+            print('Command line:', cmd_line)
+
+        self.call_sct_command(cmd_line)
+
+        image = Image(faname_path)
+        overlayList.append(image)
+        del overlayList[0]
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'gray'
+
+        image = Image(adname_path)
+        overlayList.append(image)
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'gray'
+
+        image = Image(mdname_path)
+        overlayList.append(image)
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'gray'
+
+        image = Image(rdname_path)
+        overlayList.append(image)
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'gray'
+
+#Registraction between subject imaging (T1w or T2w) with PAM50 template
+class TabPanelREG(SCTPanel):
+    DESCRIPTION = """This tool automatically performs
+    the registration between your subject and the PAM50 template. 
+    For more information, please refer to the article below.<br><br>
+    <b>Usage</b>:<br>
+    To launch the script, the uploaded images into FSLeyes must respect a sequence.
+    In the Overlay list field, the images order is, from the bottom to the top, raw imaging
+    (t1 and t2) segmentation imaging (output propseg or deepseg_sc) and disc labeling imaging (output label_utils). 
```

Comment:
i guess in auto mode you don't need to specify label_utils?  if so, please update the description accordingly